snmp_cisco/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "Cisco",
```

Comment:
I think it's something already discussed, and Natalie voted for the `Cisco` version - originally it was called `SNMP Cisco` and she told us that customers are more interested on what vendor is supported than on how they are supported. `snmp` is implementation details that can be hidden to customers - @naaltman correct me if I'm wrong!

Reply:
spot on @pducolin, snmp is the implementation detail, but customers use the integrations page to get a sense of all the different types of information we can gather from 3rd parties, whether thats a network vendor, a SaaS product, or otherwise. 

we should _mention_ that the source of the data we get is SNMP, but beyond that I don't think we need to mention it